### PR TITLE
Add privacy notice regarding data usage

### LINKED ISSUE #868: Proposed privacy consideration regarding data usage
At the 29 May teleconference [1] we discussed the usage of data from PR API for account creation. I am hearing a variety of perspectives and I am starting to think we should make a statement in the specification, for example in the security and privacy considerations. For example, something like an informative note:

Title: Data usage

Payment method owners establish the policies for how user data collected for the payment method may be used. Payment Request API sets a clear expectation that data will be used for the purposes of completing a transaction, and user experiences associated with this API convey that intention. It is the responsibility of the payee to ensure that any data usage conforms to payment method policies. For any permitted usage beyond completion of the transaction, the payee should clearly communicate that usage to the user.

[1] https://www.w3.org/2019/05/30-wpwg-minutes#item02

### PR DESCRIPTION
**Pull request by ianbjacobs**
closes #868 

The following tasks have been completed:

 * [x] Confirmed there are no ReSpec errors/warnings.
 * [ ] Has undergone security/privacy review (link)
 
Non-normative; no implementation

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/pull/869.html" title="Last updated on Jul 30, 2019, 10:51 AM UTC (3fec19e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/869/6dc6d3d...3fec19e.html" title="Last updated on Jul 30, 2019, 10:51 AM UTC (3fec19e)">Diff</a>

### CODE REVIEW

**Review by rsolomakhin — APPROVED** [review]

**Comment by ianbjacobs** [comment]
Hi @snyderp, would welcome your thoughts on this

File: index.html [hunk 1]
```diff
@@ -5209,6 +5209,21 @@ <h2>
           <a>payment method identifier</a>.
         </p>
       </section>
+      <section>
+        <h2 id="data-usage">
+          Data usage
+        </h2>
+        <p>
+          <a>Payment method</a> owners establish the policies for how user data
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          <a>Payment method</a> owners establish the privacy policies for how user data
```

**Review by marcoscaceres — COMMENTED** [review]
One small change...

**Comment by marcoscaceres** [comment]
Fixed ping to @snyderp above :)

**Comment by pes10k** [comment]
Hi @marcoscaceres, thanks for the ping!

No strong feelings from me, as long as the none of the above becomes an escape valve / reason for reducing the scrutiny applied to future payment method specs :)

**Comment by marcoscaceres** [comment]
Marcos' law of pessimism: if data is available, it will be misused and abused.

**Pull request merged**